wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/E2EIdentityCertificateDetailsView.swift

```diff
@@ -0,0 +1,143 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireCommonComponents
+
+struct E2EIdentityCertificateDetailsView: View {
+    @Environment(\.dismiss)
+    private var dismiss
+    private var borderColor: Color {
+        isDownloadAndCopyEnabled ?
+        SemanticColors.View.backgroundDefaultBlack.swiftUIColor : SemanticColors.View.backgroundDefaultWhite.swiftUIColor
+    }
```

Comment:
When it comes to border color for such a view we usually use this semantic color: backgroundSeparatorCell

Reply:
From what I understand this border added in the entire view but based on the designs we only need it on the top anchor. Can you please make the change so the border is only applied there?

Reply:
In UIKit we have a method like:

```swift
extension UIView {

    func addBorder(
        for anchor: Anchor,
        color: UIColor = SemanticColors.View.backgroundSeparatorCell,
        borderWidth: CGFloat = 1.0) {
            let border = UIView()
            addSubview(border)
            border.addConstraintsForBorder(for: anchor, borderWidth: borderWidth, to: self)
            border.backgroundColor = color
        }
```

Reply:
Done